usefulUtilities/spinnableWheel/README.md

```diff
@@ -0,0 +1,12 @@
+# Spinnable Wheel
+When you rez `spinnableWheel.json` in world, you'll see a...spinnable wheel. Click or user your hand controller laser on the wheel to spin it. Modify the User Data of the wheel and reload the Wheel's server script to update the options that appear on the wheel.

```

Comment:
lol I always love things like that

Reply:
Click or user your hand controller laser on the wheel to spin it. <-- typo?

Reply:
yes indeed, will fix :)